### PR TITLE
docs(intro): Update feature table

### PR DESCRIPTION
**Pull request by Nicell**
Updating 4 in progress features to complete! Low power sleep states, VCC shutoff, battery reporting, and one-shot keys. Also removed OLED from display support, as it's not specific to OLEDs.

### CODE REVIEW

**Comment by innovaker** [comment]
@Tsquash, I think you've been tracking this page?  Any comments?

File: docs/docs/intro.md [hunk 1]
```diff
@@ -23,16 +23,16 @@ ZMK is currently missing some features found in other popular firmware. This tab
...
+| [Display Support](features/displays)[^2]                                                                                  | 🚧  |    🚧     | ✅  |
 | [RGB Underglow](features/underglow)                                                                                       | ✅  |    ✅     | ✅  |
-| One Shot Keys                                                                                                             | 🚧  |    ✅     | ✅  |
-| Combo Keys                                                                                                                | 🚧  |           | ✅  |
+| One Shot Keys                                                                                                             | ✅  |    ✅     | ✅  |
+| [Combo Keys](https://github.com/zmkfirmware/zmk/pull/504)                                                                 | 🚧  |           | ✅  |
 | Macros                                                                                                                    | 🚧  |    ✅     | ✅  |
 | Mouse Keys                                                                                                                |     |    ✅     | ✅  |
```
Comment (Tsquash) [inline docs/docs/intro.md:31]: I think @innovaker  may be working on this? So may be in dev now.

**Review by Tsquash — APPROVED** [review]
Looks good to me. I may suggest adding a steno catagory as I see a lot of people showing interest in it in the Discord.

File: docs/docs/intro.md (hunk 1 above)
Comment (innovaker) [reply docs/docs/intro.md:31]: It's a good idea, but I don't think we're looking to officially adopt a position on steno yet.  Perhaps if the steno devs/users band together as we've previously suggested to them (it needs someone new to lead that charge).

**Comment by innovaker** [comment]
I'd suggest adding 💡 to `Mouse Keys` and `Shell over BLE`.

Might also be worth checking any updates with BlueMicro's latest.

**Comment by Nicell** [comment]
@innovaker I took a look at BlueMicro's changes and I think we're up to date.

**Review by innovaker — APPROVED** [review]

**Pull request merged**